dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json

```diff
@@ -0,0 +1,40 @@
+{
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
```

Comment:
`5s` seems like a low threshold.

Reply:
Fixed, I've bumped it to 10s, in our latest and recent results we saw a latency ~8s, so I think that's a good value to detect regressions